pkg/config/config.go

```diff
@@ -662,6 +662,8 @@ func initConfig(config Config) {
 	config.SetKnown("system_probe_config.source_excludes")
 	config.SetKnown("system_probe_config.dest_excludes")
 	config.SetKnown("system_probe_config.closed_channel_size")
+	config.SetKnown("system_probe_config.dns_timeout_in_s")
```

Comment:
Personally I would probably rather have this called: `system_probe_config.dns_timeout` and use the docs and the config template to specify the unit is seconds. Not a super strong opinion on it, but it would probably be more consistent with how other timeouts are specified through-out the agent config.

Reply:
This variable was introduced in `7.20`. I will keep your comment in mind and do the renaming whenever I do the next round of changes. But since this fix also needs to go into `7.20x`, I would rather not change code in too many places.